docs/README.md

```diff
@@ -36,4 +36,4 @@ use-lilius
 
 #### Defined in
 
-[use-lilius.ts:184](https://github.com/its-danny/use-lilius/blob/dd11a85/src/use-lilius.ts#L184)
+[use-lilius.ts:192](https://github.com/wappla/use-lilius/blob/6b366c7/src/use-lilius.ts#L192)
```

Comment:
I ran the `yarn build-docs` command. It changed theses line to my repo. Is this intended behaviour?

Reply:
@fromthemills it is, yeah. docs are just stored in the repo as markdown.

Reply:
@fromthemills altho i think it has to be generated after the release, otherwise it messes up. not really sure lol, if it's broke after merge i'll just generate it again.